lib/graphql/transfer.ts

```diff
@@ -0,0 +1,51 @@
+import { CreateTransferInput, Transfer as TransferEntry } from "./schema";
+import { Model } from "./model";
+
+export class Transfer extends Model {
+  async create(transfer: CreateTransferInput): Promise<TransferEntry> {
+    const query = `mutation {
+      createTransfer(
+        transfer: { 
+          recipient: "${transfer.recipient}", 
+          iban: "${transfer.iban}", 
+          amount: ${transfer.amount}, 
+          e2eId: "${transfer.e2eId}", 
+          note: "${transfer.note}" 
+        }
+      ) {
+        id
+        status
+        amount
+        note
+        iban
+        e2eId
```

Comment:
Does only return a confirmation_id now.

Reply:
shall we make it consistent also on the backend side?

Reply:
Unfortunately it's not yet merged to `develop`. It was done in scope of `feature/5743-mutations-fo-all-transfer-types` branch that Lera is working on.